packages/@repo/bundle-manager/package.json

```diff
@@ -46,6 +46,6 @@
     "vitest": "^3.2.3"
   },
   "engines": {
-    "node": ">=18"
+    "node": ">=20"
```

Comment:
I'd prefer we use `>=20.19`, as [vite v7 now does](https://vite.dev/blog/announcing-vite7.html#node-js-support)